extensions/vscode/src/views/homeView.ts

```diff
@@ -141,8 +141,8 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
       "activeConfigChanged",
       (cfg: Configuration | ConfigurationError | undefined) => {
         this.sendRefreshedFilesLists();
-        this.onRefreshPythonPackages();
-        this.onRefreshRPackages();
+        this.refreshPythonPackages();
+        this.refreshRPackages();
```

Comment:
renamed the `on` functions, since they no longer were functioning as event handlers.